app/services/degrees/create_from_csv_row.rb

```diff
@@ -14,7 +14,10 @@ def initialize(trainee:, csv_row:)
     end
 
     def call
-      trainee.degrees.create!(mapped_degree_attributes)
+      trainee.transaction do
+        trainee.degrees.destroy_all
+        trainee.degrees.create!(mapped_degree_attributes)
+      end
```

Comment:
Is it worth adding a test to https://github.com/DFE-Digital/register-trainee-teachers/blob/main/spec/services/degrees/create_from_csv_row_spec.rb ?